linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+title: 'Ensure That the sudo Binary Has the Correct Permissions'
+
+description: |-
+{{{ describe_file_permissions("/usr/bin/sudo", "4750") | indent(4) }}}
```

Comment:
the value of the permissions will have to be the same as in the template section